app/components/chat/Notification.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export default (props) => {
+
+  const { notifications } = props
```

Comment:
semantics: notification_**s**_ ... is this intended? Confusing based on the file name